src/helpers/ParseQuery.ts

```diff
@@ -0,0 +1,33 @@
+/**
+ * Returns what the input is (zip code, county name, or address)
+ * @Param str The string to parse
+ * @Returns A string telling what the input is
+ */
+export default function Parse(str: string): string {
+  const zip = new RegExp('^\\d{5}$');
+  // let county = new RegExp('(.+), ([a-z]{2})');
+  // let county: RegExp = /(.+), ([a-z]{2})/;
+  const county = new RegExp('(^[A-z]+), ([A-z]{2})');
+  const add1 = new RegExp('(.+), ([A-z]{2})');
+  // let add2: RegExp = /\d{1,6}\s(?:[A-Za-z0-9#]+\s){0,7}(?:[A-Za-z0-9#]+,)\s*(?:[A-Za-z]+\s){0,3}(?:[A-Za-z]+,)\s*[A-Z]{2}\s*\d{5}/;
```

Comment:
I think having the simple regex for now that can pick between the three for most cases will be good. We can expand on it later if needed